source/virtualBuffers/adobeAcrobat.py

```diff
@@ -120,3 +121,10 @@ def event_valueChange(self, obj, nextHandler):
 			return nextHandler()
 		if not self._handleScrollTo(obj):
 			return nextHandler()
+
+	def _get_ElementsListDialog(self):
+		return ElementsListDialog
+
+class ElementsListDialog(browseMode.ElementsListDialog):
+
+	ELEMENT_TYPES=(browseMode.ElementsListDialog.ELEMENT_TYPES[0],browseMode.ElementsListDialog.ELEMENT_TYPES[1])

```

Comment:
A more readable way would be to copy a slice of the list. e.g.:
ELEMENT_TYPES=browseMode.ElementsListDialog.ELEMENT_TYPES[0:2]
I.e. from 0 to 2 (exclusive)